lib/drivers/bayer/bayerContourNext.js

```diff
@@ -491,8 +491,9 @@ module.exports = function (config) {
         dataToPost.push(smbg);
       }
     } else {
-      debug('Device has no records to upload');
```

Comment:
Most of the rest of these seem to keep the `debug()` call. Should we keep it here too?

Reply:
Done.